scanner/Makefile

```diff
@@ -88,17 +89,24 @@ SCANNER_DB_INIT_BUNDLE_VERSION ?= dev
 # If true enable the DB init bundle when building the db image.
 SCANNER_DB_INIT_BUNDLE_ENABLED ?= false
 
+SCANNER_VULNERABILITY_VERSION = $(shell cat VULNERABILITY_VERSION)
```

Comment:
Curious: what happens if we do `:=`? Then this is only evaluated once, right? Curious why you chose `=` instead of `:=`

Reply:
To defer the evaluation; I don't want the file to be read for every target, unless needed.